homework5.txt

```diff
@@ -0,0 +1,224 @@
+The authenticity of host '35.228.120.217 (35.228.120.217)' can't be established.
+ECDSA key fingerprint is SHA256:dgVulx3MOrxjhF69TIw8G7TSeyV36IZMZhPpMA+o4mI.
+ECDSA key fingerprint is MD5:31:9c:84:18:91:e6:28:6f:ca:86:c0:47:b7:45:a1:ff.
+Are you sure you want to continue connecting (yes/no)? yes
+Warning: Permanently added '35.228.120.217' (ECDSA) to the list of known hosts.
+baladm@35.228.120.217's password:
+Last login: Tue Nov  6 19:08:56 2018 from 46.229.218.249
+[baladm@new-students-ssh ~]$ logout
+Connection to 35.228.120.217 closed.
+
+1.2. Generate new ssh key-pair on your localhost with name "hw-5" (keys should be created in ~/.ssh folder).
+
+1) Создаем SSH ключ (случайно не прописал новое название для ключа, оставил по дефолту):
+
+[dmitrii@localhost ~]$ ssh-keygen
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/dmitrii/.ssh/id_rsa):
+/home/dmitrii/.ssh/id_rsa already exists.
+Overwrite (y/n)? y
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Enter same passphrase again:
+Your identification has been saved in /home/dmitrii/.ssh/id_rsa.
+Your public key has been saved in /home/dmitrii/.ssh/id_rsa.pub.
+The key fingerprint is:
+SHA256:UFv1R5UzKSo/gCcYagdxOIPY2/BHvZ1BUnA++y6edDo dmitrii@localhost.localdomain
+The key's randomart image is:
++---[RSA 2048]----+
+|....o. .++=..   *|
+|..o=.....*.  o * |
+|   ==.+ oooo. o +|
+|  .ooo.+.+o+   . |
+|  . ..  S =      |
+|           +     |
+|          . +    |
+|         .E=     |
+|         .+o.    |
++----[SHA256]-----+
+
+2) Загрузка ключа на сервер:
+
+[dmitrii@localhost ~]$ ssh-copy-id baladm@35.228.120.217
+/usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/home/dmitrii/.ssh/id_rsa.pub"
+/usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed
+/usr/bin/ssh-copy-id: INFO: 1 key(s) remain to be installed -- if you are prompted now it is to install the new keys
+baladm@35.228.120.217's password:
+
+Number of key(s) added: 1
+
+Now try logging into the machine, with:   "ssh 'baladm@35.228.120.217'"
+and check to make sure that only the key(s) you wanted were added.
+
+3) Login:
+
+[dmitrii@localhost ~]$ ssh baladm@35.228.120.217
+Last login: Fri Nov 16 19:46:40 2018 from 93.92.200.170
+[baladm@new-students-ssh ~]$
+
+4)
+
+[dmitrii@localhost etc]$ cd ~/.ssh
+[dmitrii@localhost .ssh]$ ls -lh
+итого 12K
+-rw-------. 1 dmitrii dmitrii 1,7K ноя 16 20:12 id_rsa
+-rw-r--r--. 1 dmitrii dmitrii  411 ноя 16 20:12 id_rsa.pub
+-rw-r--r--. 1 dmitrii dmitrii  351 ноя 16 19:46 known_hosts
+
+5) Создаем файл config в директории ~/.ssh:
+
+[dmitrii@localhost .ssh]$ vim config
+[dmitrii@localhost .ssh]$ ssh baladm@hw5
+Bad owner or permissions on /home/dmitrii/.ssh/config
+
+/*
+Меняем права на файл
+*/
+[dmitrii@localhost .ssh]$ chmod 600 config
+[dmitrii@localhost .ssh]$ ssh baladm@hw5
+
+[dmitrii@localhost .ssh]$ ssh baladm@hw5
```

Comment:
please read task carefully, you should be able to ssh to host running exactly `ssh remotehost`